docs/docs/installation.mdx

```diff
@@ -316,3 +323,25 @@ To download a specific version, specify the version number:
 ```bash
 pip3 install rasa==3.0
 ```
+
+## M1 / M2 (Apple Silicon) Limitations
+
+Rasa installations on Apple Silicon use [Apple Metal](https://developer.apple.com/metal/) leading to reduced training times
```

Comment:
We should make metal optional, and change this

Reply:
do you mind doing the change directly on this branch?